modules/bag/tests/test_proto_conversion.h

```diff
@@ -0,0 +1,73 @@
+#pragma once
```

Comment:
i don't see any tests here, should this move to `examples`?

Reply:
This file is used only for testing. If I move it to examples it won't work.

I would probably need to create a `test_utils` module where I put this so that it can be reused, but the problem is that current CMake structure doesn't allow to create a dependency only for testing so the module would be installed everytime even when building heph as a dependency.

Now this is probably not a problem, but it is not the cleanest solution.